src/context.ts

```diff
@@ -118,15 +116,14 @@ export async function runWithParentContext<R>(
   );
 }
 
-export async function runWithDataSourceContext<R>(span: Span | undefined, callnum: number, callback: () => Promise<R>) {
+export async function runWithDataSourceContext<R>(callnum: number, callback: () => Promise<R>) {
```

Comment:
So `invokeTransactionFunction` can access the current span correctly without it being passed around now, thanks to the later `context.with`?